Modules/Loadable/Markups/VTKWidgets/vtkSlicerROIRepresentation2D.h

```diff
@@ -111,6 +111,8 @@ class VTK_SLICER_MARKUPS_MODULE_VTKWIDGETS_EXPORT vtkSlicerROIRepresentation2D :
   vtkSmartPointer<vtkProperty2D>       ROIOutlineProperty;
   vtkSmartPointer<vtkActor2D>          ROIOutlineActor;
 
+  vtkSmartPointer<vtkCellLocator>      ROIOutlineLocator;
```

Comment:
Do we really need a locator? Since we only have a handful of lines, iterating through them should be much quicker than maintaining an octree and looking up cells in an octree.